switchapi/switch_handle.h

```diff
@@ -0,0 +1,204 @@
+/*
+Copyright 2013-present Barefoot Networks, Inc.
```

Comment:
We don't use Barefoot license anymore, just apache is good enough. If you check latest files, that should have it. We can take care of it as part of my next commit as well

Reply:
The next commit is the right time to do it.